api/app/signals/apps/email_integrations/integrations/core.py

```diff
@@ -1,124 +1,76 @@
 """
 E-mail integration for 'core' Signal behaviour.
 """
-import logging
-import re
-
-import pytz
 from django.conf import settings
 from django.core.mail import send_mail
 from django.template import loader
-from django.utils import timezone
 
 from signals.apps.email_integrations.messages import ALL_AFHANDELING_TEXT
-from signals.apps.signals.workflow import AFGEHANDELD
+from signals.apps.signals import workflow
+
+
+def send_mail_reporter_created(signal):
+    """Send a notification e-mail to the reporter about initial create of the given `Signal` object.
+
+    :param signal: Signal object
+    :returns: number of successfully send messages or None
+    """
+    if not signal.reporter.email:
+        return None
+
+    subject = f'Bedankt voor uw melding ({signal.id})'
+    message = create_initial_create_notification_message(signal)
+    to = signal.reporter.email
 
-LOG = logging.getLogger()
+    return send_mail(subject, message, settings.NOREPLY, (to, ))
 
 
-# TODO: fetch PDF and attach to message?
+def create_initial_create_notification_message(signal):
+    """Create e-mail body message about initial create of the given `Signal` object.
 
+    :param signal: Signal object
+    :returns: message (str)
+    """
+    context = {
+        'signal': signal,
+        'afhandelings_text': (
+            ALL_AFHANDELING_TEXT[signal.category_assignment.sub_category.handling]
+        ),
+    }
+    template = loader.get_template('email/signal_created.txt')
+    message = template.render(context)
+    return message
 
-def get_valid_email(signal):
-    email_valid = r'[^@]+@[^@]+\.[^@]+'
-    if signal.reporter and signal.reporter.email and re.match(email_valid, signal.reporter.email):
-        return signal.reporter.email
-    else:
+
+def send_mail_reporter_status_changed(signal, status):
+    """Send a notification e-mail to the reporter about status change of the given `Signal` object.
```

Comment:
(also below) Possibly change function name to reflect that is is only status change to AFGEHANDELD that we mail about.

Reply:
done